appveyor.yml

```diff
@@ -24,7 +24,10 @@ build_script:
   - conda update conda
   - conda install conda-build
   - ps: if(-not $env:APPVEYOR_PULL_REQUEST_NUMBER) { conda install anaconda-client }
-  - conda build conda.recipe --channel dynd/channel/dev
+  - conda build conda\ndt.recipe --channel dynd/channel/dev
+  - rd /q /s build dist dynd.ndt.egg-info
+  - echo dynd.ndt >> requirements.txt
```

Comment:
Okay, so, thinking about this a bit more, I'd really like to avoid automatic edits to committed files that get run as a part of the build process. That's a recipe for accidental garbage getting put into the repo. Is there any possible way around that?